Automattic/sensei
# Move blocks title, description, keywords to `block.json` and fix localization

This change attempts to fix the missing blocks title/description in https://wordpress.org/plugins/sensei-lms/

### Changes proposed in this Pull Request

* Make sure all blocks have a title and description in their block.json.

### Testing instructions

* All blocks should work as before.
* Make sure the blocks title and description are translatable by [following the steps in this comment](https://github.com/Automattic/sensei/pull/5782#issuecomment-1265911613).
